lib/Page.js

```diff
@@ -50,7 +50,7 @@ class Page extends EventEmitter {
       client.send('Target.setAutoAttach', {autoAttach: true, waitForDebuggerOnStart: false}),
       client.send('Page.setLifecycleEventsEnabled', { enabled: true }),
       client.send('Network.enable', {}),
-      client.send('Runtime.enable', {}),
+      client.send('Runtime.enable', {}).then(() => page._frameManager.ensureSecondaryDOMWorld()),
```

Comment:
why does this have to happen after runtime.enable?

Reply:
All execution contexts will be reported before Runtime.enable returns. If there's the one with helper isolated world, it'll be reported.